tilequeue/query/__init__.py

```diff
@@ -1,4 +1,9 @@
-from tilequeue.query import postgres, fixture
+from tilequeue.query.fixture import make_data_fetcher
+from tilequeue.query.pool import DBConnectionPool
+from tilequeue.query.postgres import make_db_data_fetcher
 
-make_db_data_fetcher = postgres.make_db_data_fetcher
-make_fixture_data_fetcher = fixture.make_data_fetcher
+__all__ = [
+    'DBConnectionPool',
+    'make_data_fetcher',
```

Comment:
Doh, sorry I missed that. Is the rename important? The tests pass for me locally without it.

If we do want the rename, I don't think `__all__` can do it and we should just set a variable at module scope like you had it before for that particular import.